### PR TITLE
anantomical reference

### LINKED ISSUE #926: adding reference_manipulator_coordinates and target_manipulator_coordinates
**Is your feature request related to a problem? Please describe.**
We usually have two manipulator coordinates corresponding to the target and reference coordinates during the experiment. It's meaningless to have manipulator coordinates without providing the reference manipulator coordinates. Currently, we only have the `manipulator_coordinates` in the `session.py/ManipulatorModule`.
**Describe the solution you'd like**
Current
image

Suggested
image

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by saskiad**
closes #926

### CODE REVIEW

**Comment by saskiad** [comment]
more error messages that don't make sense

**Comment by jtyoung84** [comment]
> more error messages that don't make sense

I'll look into it today. We're moving the automated schema version bumps whenever `main` is updated. I'll remove those checks when a PR is opened in `dev`. I think that's what those error messages are related to.

File: src/aind_data_schema/core/session.py [hunk 1]
```diff
@@ -388,7 +391,7 @@ class Stream(AindModel):
     fiber_connections: List[FiberConnectionConfig] = Field(default=[], title="Implanted fiber photometry devices")
     fiber_modules: List[FiberModule] = Field(default=[], title="Inserted fiber modules")
     ophys_fovs: List[FieldOfView] = Field(default=[], title="Fields of view")
-    slap_fovs: List[SlapFieldOfView] = Field(default=[], title="Slap2 fields of view")
+    slap_fovs: Optional[SlapFieldOfView] = Field(default=None, title="Slap2 field of view")
```
Comment (dyf) [inline src/aind_data_schema/core/session.py:394]: this should go back to being a list, I think

File: src/aind_data_schema/core/session.py [hunk 2]
```diff
@@ -208,7 +208,7 @@ class ManipulatorModule(DomeModule):
     """A dome module connected to a 3-axis manipulator"""
 
     primary_targeted_structure: str = Field(..., title="Targeted structure")
-    other_targeted_structure: Optional[List[str]] = Field(default=None, title="Other targeted structure")
+    other_targeted_structure: Optional[List[str]] = Field(None, title="Other targeted structure")
```
Comment (dyf) [inline src/aind_data_schema/core/session.py:211]: bring `default=` back

File: src/aind_data_schema/core/session.py [hunk 3]
```diff
@@ -217,11 +217,14 @@ class ManipulatorModule(DomeModule):
         ...,
         title="Manipulator coordinates",
     )
-    bregma_coordinates: Optional[Coordinates3d] = Field(default=None, title="Bregma coordinates")
-    surface_z: Optional[Decimal] = Field(default=None, title="Surface z")
+    anatomical_coordinates: Optional[Coordinates3d] = Field(None, title="Anatomical coordinates relative to reference")
```
Comment (dyf) [inline src/aind_data_schema/core/session.py:220]: `default=`

File: src/aind_data_schema/core/session.py [hunk 4]
```diff
@@ -481,12 +484,12 @@ def validate_stream_modalities(cls, value: List[Modality.ONE_OF], info: Validati
 class StimulusEpoch(AindModel):
     """Description of stimulus used during session"""
 
-    stimulus_start_time: AwareDatetimeWithDefault = Field(
+    stimulus_start_time: datetime = Field(
```
Comment (dyf) [inline src/aind_data_schema/core/session.py:487]: we want the timezone-aware version, right?

File: src/aind_data_schema/core/session.py [hunk 5]
```diff
@@ -582,7 +585,7 @@ class Session(AindCoreModel):
     headframe_registration: Optional[Affine3dTransform] = Field(
         None, title="Headframe registration", description="MRI transform matrix for headframe"
     )
-    reward_delivery: Optional[RewardDeliveryConfig] = Field(default=None, title="Reward delivery")
-    reward_consumed_total: Optional[Decimal] = Field(default=None, title="Total reward consumed (mL)")
-    reward_consumed_unit: VolumeUnit = Field(VolumeUnit.ML, title="Reward consumed unit")
-    notes: Optional[str] = Field(default=None, title="Notes")
+    reward_delivery: Optional[RewardDeliveryConfig] = Field(None, title="Reward delivery")
+    reward_consumed_total: Optional[Decimal] = Field(None, title="Total reward consumed (uL)")
+    reward_consumed_unit: VolumeUnit = Field(VolumeUnit.UL, title="Reward consumed unit")
```
Comment (jtyoung84) [inline src/aind_data_schema/core/session.py:590]: This is changing the `reward_consumed_unit` from `ML` to `UL`

**Comment by saskiad** [comment]
Closing this because something's really wonky with it. Redone with #1013

**Pull request closed without merging**